src/app/logic/defaultGameConfiguration.ts

```diff
@@ -0,0 +1,136 @@
+        { diceNumber: 2, probability: 6 },
+      ],
+    },
+    {
+      name: AnimalNames.SHEEP,
+      tradeValue: 6,
+      path: '/static/images/avatars/sheep.png',
+      role: AnimalRoles.LIVESTOCK,
+      playersInitialStock: 0,
+      bankInitialStock: 24,
+      dice: [
+        { diceNumber: 1, probability: 2 },
+        { diceNumber: 2, probability: 3 },
+      ],
+    },
+    {
+      name: AnimalNames.PIG,
+      tradeValue: 12,
+      path: '/static/images/avatars/pig.png',
+      role: AnimalRoles.LIVESTOCK,
+      playersInitialStock: 0,
+      bankInitialStock: 20,
+      dice: [
+        { diceNumber: 1, probability: 2 },
+        { diceNumber: 2, probability: 1 },
+      ],
+    },
+    {
+      name: AnimalNames.COW,
+      tradeValue: 36,
+      path: '/static/images/avatars/cow.png',
+      role: AnimalRoles.LIVESTOCK,
+      playersInitialStock: 0,
+      bankInitialStock: 12,
+      dice: [{ diceNumber: 2, probability: 1 }],
+    },
+    {
+      name: AnimalNames.HORSE,
+      tradeValue: 72,
+      path: '/static/images/avatars/horse.png',
+      role: AnimalRoles.LIVESTOCK,
+      playersInitialStock: 0,
+      bankInitialStock: 4,
+      dice: [{ diceNumber: 1, probability: 1 }],
+    },
+  ],
+  protectorsConfig: [
+    {
+      name: AnimalNames.SMALL_DOG,
+      tradeValue: 6,
+      path: '/static/images/avatars/dog.png',
+      role: AnimalRoles.GUARDIAN,
+      playersInitialStock: 0,
+      bankInitialStock: 4,
+      chasesAway: AnimalNames.FOX,
+    },
+    {
+      name: AnimalNames.BIG_DOG,
+      tradeValue: 36,
+      path: '/static/images/avatars/dog.png', // TODO: CHANGE TO BIG/SMALL DOG
+      role: AnimalRoles.GUARDIAN,
+      playersInitialStock: 0,
+      bankInitialStock: 2,
+      chasesAway: AnimalNames.WOLF,
+    },
+  ],
+  predatorsConfig: [
+    {
+      name: AnimalNames.FOX,
+      path: '/static/images/avatars/fox.png',
+      kills: [AnimalNames.RABBIT],
+      isChasedAwayBy: [AnimalNames.SMALL_DOG],
+      dice: [{ diceNumber: 1, probability: 1 }],
+    },
+    {
+      name: AnimalNames.WOLF,
+      path: '/static/images/avatars/wolf.png',
+      kills: [
+        AnimalNames.RABBIT,
+        AnimalNames.SHEEP,
+        AnimalNames.PIG,
+        AnimalNames.COW,
+      ],
+      isChasedAwayBy: [AnimalNames.BIG_DOG],
+      dice: [{ diceNumber: 2, probability: 1 }],
+    },
+  ],
+};
+
+// diceConfig: [
+//   [
+//     { name: AnimalNames.RABBIT, probability: 6 },
+//     { name: AnimalNames.SHEEP, probability: 2 },
+//     { name: AnimalNames.PIG, probability: 2 },
+//     { name: AnimalNames.HORSE, probability: 1 },
+//     { name: AnimalNames.FOX, probability: 1 },
+//   ],
+//   [
+//     { name: AnimalNames.RABBIT, probability: 6 },
+//     { name: AnimalNames.SHEEP, probability: 3 },
+//     { name: AnimalNames.PIG, probability: 1 },
+//     { name: AnimalNames.COW, probability: 1 },
+//     { name: AnimalNames.FOX, probability: 1 },
+//   ],
+// ],
```

Comment:
would You like to use this code from comments?

Reply:
This code is supposed to be discussed when it comes to the way we create dice objects